SPECS-EXTENDED/389-ds-base/389-ds-base.spec

```diff
@@ -732,6 +732,9 @@ exit 0
 %endif
 
 %changelog
+* Tue Apr 01 2025 Kavya Sree Kaitepalli <kkaitepalli@microsoft.com> 3.1.0-3
```

Comment:
We should also bump cloud-hypervisor-cvm.

For kata-containers, kata-containers-cc we need to upgrade our sources. Hence, we need to not upgrade the minor version to avoid a re-build. but likely we should version-pin those two packages to the prior version

Reply:
For kata-containers, kata-containers-cc: "BuildRequires:  rust <= 1.75.0"